#include<iostream>
using namespace std;

int main() {

    cout << "Before Exception" << endl;
    cout << 2/0;
    cout << "After Exception" << endl;

    return 0;
}